decidim-core/app/helpers/decidim/safe_params_helper.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A helper to allow only Decidim params to be added to a link.
+  # This is useful when we want to preserve the params from a seArch, ordering
+  # or paginating results. Using this, we can link back to where the user was
+  # at the show page.
+  module SafeParamsHelper
```

Comment:
Shouldn't it be `FilterParamsHelper`?
With "SafeParams" it seems to me that it does a sanitization to any params.
```suggestion
  module FilterParamsHelper
```